src/API/HealthController.ts

```diff
@@ -39,13 +66,49 @@ export class HealthController {
     }
   }
 
+  private async getBlockchainInfo(): Promise<any> {
+    try {
+      const { blockchainInfo } = (await this.collection.findOne({ name: 'blockchainInfo' })) || {
+        blockchainInfo: 'Getting Blockchain Info...',
+      }
+      return blockchainInfo
+    } catch (e) {
+      return { blockchainInfo: 'Error retrieving blockchainInfo...' }
```

Comment:
I think returning blockchainInfo inside an object is going to result in a nested blockchainInfo

```ts
{
  blockchainInfo: {
    blockchainInfo: 'Error retrieving blockchainInfo...'
  }
}
```

We can just change this key to error like:

```ts
return { error: 'Error retrieving blockchainInfo...' }
```

So that it puts an error prop instead

```ts
{
  blockchainInfo: {
    error: 'Error retrieving blockchainInfo...'
  }
}
```

Reply:
If you agree, apply to the other 2 functions as well

Reply:
Ok yeah. I was thinking I was destructuring blockchainInfo from the returned value of this function, but I'm not.